contracts/Optimist.sol

```diff
@@ -1,6 +0,0 @@
-pragma solidity ^0.4.25;
-
-interface Optimist {
```

Comment:
Why was this interface removed?

Reply:
I started with writing that interface but didn't come around inheriting from it. In an interface all declared functions must be external, whereas I wanted to have internal functions in `mintNotes`, `spendNotes` and `liquidateNotes`.

Reply:
Sounds good